homeworks/artem.bilenko_brilux/homework_6/homework_6_task_1.js

```diff
@@ -0,0 +1,38 @@
+function User() {
+    let fullName;
+    let phone;
+
+    Object.defineProperty(this, 'fullName', {
+        get() {
+            return fullName;
+        },
+        set(editedName) {
+            fullName = editedName
+                .toLowerCase()
+                .replace(/\b\w/g, l => l.toUpperCase());
+        },
+    });
+    Object.defineProperty(this, 'phone', {
+        get() {
+            return phone;
+        },
+        set(editedPhone) {
+            const newPhone = editedPhone.replace(/[^+\d]/g, '');
+            if (newPhone[0] === '+') {
+                newPhone.replace(/[^\d]/g, '');
+                newPhone[0] = '+';
+                phone = newPhone;
+            } else {
+                phone = newPhone.replace(/[^\d]/g, '');
+            }
```

Comment:
using almost same code 3 times in 7 lines... please rebuild so it is used only once, especially RegExp